core-drawer-panel.html

```diff
@@ -185,60 +189,66 @@
         this.selected = this.defaultSelected;
       }
       this.narrow = this.queryMatches;
-      this.setAttribute('touch-action', 
+      this.setAttribute('touch-action',
           this.narrow && !this.disableSwipe ? 'pan-y' : '');
       this.fire('core-responsive-change', {narrow: this.narrow});
     },
-    
+
     // swipe support for the drawer, inspired by
     // https://github.com/Polymer/core-drawer-panel/pull/6
     trackStart : function(e) {
-      if (this.narrow && !this.disableSwipe) {
-        this.dragging = true;
-
-        if (this.selected === 'main') {
-          this.dragging = this.rightDrawer ?
-              e.pageX >= this.offsetWidth - this.edgeSwipeSensitivity :
-              e.pageX <= this.edgeSwipeSensitivity;
-        }
-
-        if (this.dragging) {
-          this.width = this.$.drawer.offsetWidth;
-          this.transition = false;
-          e.preventTap();
-        }
+      if (!this.narrow && this.disableSwipe) {
```

Comment:
When negating a compound statement, the operator changes from AND to OR (or vice versa) according to [De Morgan's laws](https://en.wikipedia.org/wiki/De_Morgan%27s_laws). `!(a && !b)` is equivalent to `(!a || b)`.

This should be:

```js
if (!this.narrow || this.disableSwipe)
```